packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -74,6 +189,80 @@ function getFrameworkTests(opts: {
 				},
 				unsupportedOSs: ["win32"],
 			},
+			nuxt: {
+				quarantine: true,
```

Comment:
just double checking, this is still supposed to quarantined?

Reply:
It fails in this PR - but then passes in after the Nuxt PR 😄